src/lib/y2storage/autoinst_profile/drive_section.rb

```diff
@@ -302,6 +303,18 @@ def use_value_from_string(use)
         return use unless use =~ /(\d+,?)+/
         use.split(",").select { |n| n =~ /\d+/ }.map(&:to_i)
       end
+
+      # Determine whether snapshots are enabled for the current drive
+      #
+      # Currently AutoYaST does not support enabling/disabling snapshots
+      # for a partition but for the whole disk.
+      #
+      # @param disk [Y2Storage::Disk, Y2Storage::Dasd] Disk
+      # @return [Boolean] true if snapshots are enabled
+      def enabled_snapshots?(disk)
+        filesystems = disk.partitions.map(&:filesystem)
```

Comment:
I guess we don't have to worry here about the case of a disk directly formatted (no partitions)